Docs 8655 provision script/template info (#274)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-azure.adoc

== Installation

[NOTE]

> The Azure Resource Manager templates provided as part of the installation package are for reference only and should not be used as-is for either non-production or production Runtime Fabric installations. You must review and customize these templates to meet your needs. Runtime Fabric is supported as long as you meet or exceed the minimum requirements specified in xref:install-prereqs.adoc[Runtime Fabric Installation Prerequisites].

Reviewer: ```suggestion
Do not use the Azure Resource Manager templates provided in the installation package as-is for any Runtime Fabric installation. Review and customize these templates to meet your needs. Runtime Fabric is supported as long as you meet the requirements specified in xref:install-prereqs.adoc[Runtime Fabric Installation Prerequisites].
```
